api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -187,7 +187,6 @@ message ExternalProcessor {
   // field is set in an external processor response.
   bool disable_clear_route_cache = 11;
 
-  // [#not-implemented-hide:]
   // Allow headers matching the ``forward_rules`` to be forwarded to the external processing server.
   // If not set, all headers are forwarded to the external processing server.
   HeaderForwardingRules forward_rules = 12;
```

Comment:
Does a disallow list make sense? 
often times we are doing filtering based on blacklist: remove some internal headers (e.g. x-envoy-xxx)  and sent everything else to the external service.

Reply:
Should we enforce allow list or disallow list is discussed in this PR:
https://github.com/envoyproxy/envoy/pull/27333